cloud.foundry.cli/src/main/java/cloud/foundry/cli/services/GetController.java

```diff
@@ -21,35 +19,36 @@
 import java.util.Map;
 import java.util.concurrent.Callable;
 
-
 /**
- * This class realizes the functionality that is needed for the get commands. They provide various information about a
- * cloud foundry instance.
+ * This class realizes the functionality that is needed for the get commands.
+ * They provide various information about a cloud foundry instance.
  */
-@Command(name = "get",
-        header = "%n@|green Get the current configuration of your cf instance.|@",
-        mixinStandardHelpOptions = true,
-        subcommands = {
-                GetController.GetServicesCommand.class,
-                GetController.GetSpaceDevelopersCommand.class,
-                GetController.GetApplicationsCommand.class,
-                GetController.GetAllInformation.class})
+@Command(name = "get", header = "%n@|green Get the current configuration of your cf instance.|@",
+    mixinStandardHelpOptions = true,
+    subcommands = {
+    GetController.GetServicesCommand.class,
+    GetController.GetSpaceDevelopersCommand.class,
+    GetController.GetApplicationsCommand.class
+})
 public class GetController implements Callable<Integer> {
 
+    @Mixin
+    private static LoginCommandOptions loginOptions;
+
     @Override
     public Integer call() throws Exception {
-        //TODO: Make this by default return all information (convenient shortcut)
-        usage(this, System.out);
+        DefaultCloudFoundryOperations cfOperations = CfOperationsCreator.createCfOperations(loginOptions);
+        GetLogic getLogic = new GetLogic(cfOperations);
+        Log.info("Fetching all information for target space...");
+        ConfigBean allInformation = getLogic.getAll();
+
+        System.out.println(YamlMapper.dump(allInformation));
```

Comment:
We should not copy the code from the get all command. Instead we should call the get all command within this method (e.g. by adding a constructor to GetAllInformation and creating an instance that is called).